tooling/src/hypothesistooling/releasemanagement.py

```diff
@@ -137,7 +137,7 @@ def parse_release_file(filename):
 
 
 def parse_release_file_contents(release_contents, filename):
-    release_lines = release_contents.split('\n')
+    release_lines = [l.rstrip() for l in release_contents.split('\n')]
```

Comment:
Worth a test? (Look in `test_release_management.py`.)

Reply:
Sure, I'll push at the other end of my flight. Might add Bandit too, since it should be trivial and needs to go in soon.